src/tabular/encoder.ts

```diff
@@ -0,0 +1,121 @@
+import { Tensor, Tensor1D } from "@tensorflow/tfjs-core";
+import { Series } from "danfojs-node";
+import { _zeros, getDataByType } from '../utils';
+
+/**
+ * Encodes an array, Tensor or Danfo Series using unique labels
+ * @param {data} data [Array|Series]
+ * @returns Array
+ */
+class LabelEncoder {
+  public labels: Array<any>
+  private _data: Tensor1D[] | Array<any> | Series | any
```

Comment:
```suggestion
  private _data: Tensor1D[] | Array<any> | Series
```

Reply:
@yorkie Removing the any gives the following compilation warning and the code fails to compile:

> The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.ts(2358)